### DISCUSSION TITLE
Ошибка: файл слишком большой

### DISCUSSION THREAD

**Discussion by gusin-itas** [totumonline/totum-ru-issues-and-discussions#531] [ОШБК!] [answered]
Не получается загружать в систему файлы размером более 1 мегабайта. Менял переменные Php по рекомендации https://ru.docs.totum.online/debugging#sx-maximum-upload (подправьте там название post_maxsize на правильное post_max_size), перезапускал службы php-fpm и nginx. Добавил для проверки в каталог тотума файлик info.php с вызовом phpinfo() - изменения происходят, только это не помогает загружать в тотум файлы большего, чем 1М размера.
upload_max_filesize сделал 20М, post_max_size - 30М, но всё без толку...

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Что за установка? Скриптом установки поставлено? Обновлено?

тк 1 мб это стандарт nginx при проксировании.

В /etc/nginx/sites-available/totum.conf есть строка client_max_body_size 10M; в секции сервера перед location?

**Reply by gusin-itas** [reply]
да было поставлено из скрипта, но использовался обратный прокси nginx и вот там как раз этой строки не было в настройках, добавил и всё замечательно заработало!
благодарю от души за подсказку!

**Reply by ttm-alexeypolunin** [reply]
Ок! Верните только в настройке пула php_admin_value[post_max_size] = 0 это его отключает.

В доках в том месте я поправлю (забыл, что оно вообще существует там).
